control/control_performance_analysis/include/control_performance_analysis/control_performance_analysis_core.hpp

```diff
@@ -89,6 +90,7 @@ class ControlPerformanceAnalysisCore
   uint odom_interval_;
   double acceptable_min_waypoint_distance_;
   double prevent_zero_division_value_;
+  double lpf_gain;
```

Comment:
Member variables should be suffixed by an underscore.
```suggestion
  double lpf_gain_;
```